docs/tildagon-apps/development.md

```diff
@@ -36,6 +36,12 @@ class ExampleApp(app.App):
 __app_export__ = ExampleApp
 ```
 
+If you place this code in a file called `app.py`, you may also need to add a separate file called `__init__.py` containing this line:
+
+```python
+from .app import ExampleApp
```

Comment:
This is not necessary. If the app isn't running without this on your badge, that's a bug. The format for an app does not include the __init__.py file, so we would fix it by identifying why your badge requires that.